crates/eframe/src/native/epi_integration.rs

```diff
@@ -278,13 +278,23 @@ impl EpiIntegration {
 
         let full_output = self.egui_ctx.run(raw_input, |egui_ctx| {
             if let Some(viewport_ui_cb) = viewport_ui_cb {
-                // Child viewport
+                // Child Deferred Viewport
                 crate::profile_scope!("viewport_callback");
                 viewport_ui_cb(egui_ctx);
             } else {
+                // ROOT Viewport ( with Immediate Viewport )
                 crate::profile_scope!("App::update");
                 app.update(egui_ctx, &mut self.frame);
             }
+
+            let continuous_mode = egui_ctx.options(|options| options.continuous_mode);
+            if continuous_mode {
+                let viewport_id = egui_ctx.viewport_id();
+                // TODO(rustbasic) : Do not recall until the next repaint.
+                // 1000 millis / 60 fps = 16.67 millis
+                self.egui_ctx
+                    .request_repaint_after_for(std::time::Duration::from_millis(8), viewport_id);
```

Comment:
why 8 ms delay?

Reply:
> why 8 ms delay?

It takes about two attempts.
It is the most efficient.
If you do 16.67ms, 60 fps cannot be achieved.

Reply:
But why not request an immediate repaint? eframe has vsync anyways

Reply:
> But why not request an immediate repaint? eframe has vsync anyways



> But why not request an immediate repaint? eframe has vsync anyways

This is an unnecessary waste of CPU usage.
It would be a good idea to apply this and continue to monitor and upgrade.
It can't be perfect from the beginning.

Reply:
It's not a waste of CPU, because of vsync.